invenio_vocabularies/contrib/awards/awards.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Vocabulary awards."""
+
+from invenio_records_resources.factories.factory import RecordTypeFactory
+
+from ...records.pidprovider import PIDProviderFactory
+from ...records.systemfields import BaseVocabularyPIDFieldContext
+from ...services.permissions import PermissionPolicy
+from .config import AwardsSearchOptions, service_components
+from .schema import AwardSchema
+
+record_type = RecordTypeFactory(
```

Comment:
major: it's missing the relation ("_programmatic fk_"), therefore  it is not being linked to a funder. See [names](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/contrib/names/names.py#L23) as an example. This also means that when doing an award test the funder needs to be in the DB (i.e. use both funder and award fixture -  see names)